withdraw-by-check/REPWRITERSPECS/BANNO.CHECK.WITHDRAW.V1.POW

```diff
@@ -1061,5 +1166,387 @@ PROCEDURE BNODEBUGMSGADD
+ NEWLINE
+
+ TMPCHR=FORMAT("########9.99",PARAMMINWDLIMIT)
+ CALL NLTS
+ PRINT ","
+ JSONPROPNAME="minWdAmount"
+ CALL JSONPROP
+ PRINT Q+TMPCHR+Q
+ NEWLINE
+
+ TMPCHR=FORMAT("########9.99",PARAMMAXWDLIMIT)
+ CALL NLTS
+ PRINT ","
+ JSONPROPNAME="maxWdAmount"
+ CALL JSONPROP
+ PRINT Q+TMPCHR+Q
+ NEWLINE
+
+ PRINT ","
+ JSONPROPNAME="owner"
+ CALL JSONPROP
+ PRINT Q+NAME:LONGNAME+Q
+ NEWLINE
+
+ PRINT ","
+ JSONPROPNAME="address"
+ CALL JSONPROP
+ NEWLINE
+ CALL JSONARRAY
+ TMPCHR=ACCOUNT:PAYEELINE:1
+ CALL NLTS
+ PRINT Q+TMPCHR+Q+","
+ NEWLINE
+ TMPCHR=ACCOUNT:PAYEELINE:2
+ CALL NLTS
+ PRINT Q+TMPCHR+Q+","
+ NEWLINE
+ TMPCHR=ACCOUNT:PAYEELINE:3
+ CALL NLTS
+ PRINT Q+TMPCHR+Q+","
+ NEWLINE
+ TMPCHR=ACCOUNT:PAYEELINE:4
+ CALL NLTS
+ PRINT Q+TMPCHR+Q+","
+ NEWLINE
+ TMPCHR=ACCOUNT:PAYEELINE:5
+ CALL NLTS
+ PRINT Q+TMPCHR+Q+","
+ NEWLINE
+ TMPCHR=ACCOUNT:PAYEELINE:6
+ CALL NLTS
+ PRINT Q+TMPCHR+Q+""
+ NEWLINE
+ CALL JSONARRAYEND [end address array]
+
+ PRINT ","
+ JSONPROPNAME="disclaimerText"
+ CALL JSONPROP
+ NEWLINE
+ CALL JSONARRAY
+ FOR TMPLOOP=1 TO PARAMCUSTOMTERMLINES
+  DO
+   TMPCHR=PARAMCUSTOMTERMS(TMPLOOP)
+   CALL NTS
+   PRINT Q+TMPCHR+Q
+   IF TMPLOOP<PARAMCUSTOMTERMLINES THEN
+    PRINT ","
+   NEWLINE
+  END
+ CALL JSONARRAYEND [end disclaimerText array]
+ 
+ CALL JSONOBJEND [results object]
+END [PROCEDURE]
+
+PROCEDURE JSONTRANSUCCESS
+
+ JSONPROPNAME="results"
+ CALL JSONPROP
+ CALL JSONOBJ
+ NEWLINE
+
+ JSONPROPNAME="success"
+ CALL JSONPROP
+ PRINT JSONBOOLEAN(TRUE)
+ NEWLINE
+
+ PRINT ","
+ JSONPROPNAME="memoMode"
+ CALL JSONPROP
+ PRINT JSONBOOLEAN(SYSMEMOMODE)
+ NEWLINE
+
+ CALL JSONOBJEND [results object]
+END [PROCEDURE]
+
+PROCEDURE JSONTRANFAILED
+
+ CALL BNOERRGET
+
+ JSONPROPNAME="errorCode"
+ CALL JSONPROP
+ PRINT FORMAT("999",BNOERRCODE)
+ NEWLINE
+
+ PRINT ","
+ JSONPROPNAME="loggingErrorMessage"
+ CALL JSONPROP
+ PRINT QQ+BNOERRMESSAGE+QQ
+ NEWLINE
+
+ IF BNOERRDISPLAYMSGCOUNT>0 THEN
+  DO
+   PRINT ","
+   JSONPROPNAME="displayErrorMessage"
+   CALL JSONPROP
+
+   CALL JSONARRAY
+   FOR TMPLOOP=1 TO BNOERRDISPLAYMSGCOUNT
+    DO
+     TMPCHR=BNOERRDISPLAYMSG(TMPLOOP)
+     CALL NTS
+     PRINT Q+TMPCHR+Q
+     IF TMPLOOP<BNOERRDISPLAYMSGCOUNT THEN
+      PRINT ","
+     NEWLINE
+    END
+   CALL JSONARRAYEND
+  END
+
+ IF TRANSTATUS=TRANLIMITS AND
+    BNOERRCODE=BNOERRCODEINVALIDAMT THEN
+  DO
+   PRINT ","
+   TMPCHR=FORMAT(" ###,##9.99",RGWDAMOUNT)
+   CALL NLS
+   JSONPROPNAME="requested"
+   CALL JSONPROP
+   PRINT Q+TMPCHR+Q
+   NEWLINE
+
+   PRINT ","
+   TMPCHR=FORMAT(" ###,##9.99",PARAMMINWDLIMIT)
+   CALL NLS
+   JSONPROPNAME="minWdAmount"
+   CALL JSONPROP
+   PRINT Q+TMPCHR+Q
+   NEWLINE
+
+   PRINT ","
+   TMPCHR=FORMAT(" ###,##9.99",PARAMMAXWDLIMIT)
+   CALL NLS
+   JSONPROPNAME="maxWdAmount"
+   CALL JSONPROP
+   PRINT Q+TMPCHR+Q
+   NEWLINE
+  END 
+END [PROCEDURE]
+
+[*
+**  --------------------------------------------------------------------
+**  JSON Procedures - Common
```

Comment:
You have a few procedures in this section of the program (JSON procedures) which just have one line of code (eg: PROCEDURE JSONOBJ). Does this add unnecessary stress on the system to divert program run to a procedure to run one line of code... instead of just running that one line of code from the original location?

Reply:
I would not think so.  I think the benefit outweighs any concern in those regards.  Using those procedures just standardizes the way json objects, arrays, etc. are opened and closed and prevents typos.  That's my take...

Reply:
works for me then